crates/bevy_ui/src/ui_node.rs

```diff
@@ -1021,6 +1021,10 @@ pub enum MinTrackSizingFunction {
     MaxContent,
     /// Track minimum size should be automatically sized
     Auto,
+    /// Track minimum size should be a percent of the viewport's smaller dimension.
```

Comment:
These doc comments should ideally describe what this value does on the first line, and then add this information in a second line :)

Reply:
Sorry Alice, I don't _quite_ understand what you're saying here